generators/entity-server/templates/src/main/java/package/domain/Entity.java.jhi.spring_data_couchbase.ejs

```diff
@@ -173,7 +174,7 @@ import java.util.stream.Collectors;
         this.<%= relationship.relationshipFieldNamePlural %>.add(<%= relationship.otherEntityName %>);
       <%_ if (!relationship.otherEntity.embedded || embedded && relationship.ownerSide) { _%>
         <%_ if (!relationship.otherEntity.embedded) { _%>
-        this.<%= relationship.relationshipFieldName %>Ids.add(<%= relationship.otherEntityName %>.get<%= primaryKey.nameCapitalized %>());
+        this.<%= relationship.relationshipFieldName %>Ids.add(<%= relationship.otherEntityName %>.getId());
```

Comment:
The reason I switched it to `Id` is because after one of the rebases (I think it was the one where the Entity.java.ejs file was split up into multiple files by database), I was getting an error when I try to generate the app that pertained to the `primaryKey`. Will try to change it back and debug to see if I'm still getting that error.

Reply:
This is the error I'm getting:
```
    175| <%_ if (!relationship.otherEntity.embedded || embedded && relationship.ownerSide) { _%>
    176| <%_ if (!relationship.otherEntity.embedded) { _%>
 >> 177|         this.<%= relationship.relationshipFieldName %>Ids.add(<%= relationship.otherEntityName %>.get<%= primaryKey.nameCapitalized %>());
    178| <%_ } _%>
    179| <%_ if (relationship.otherRelationship) { _%>
    180| <%_ if (!relationship.otherRelationship.collection) { _%>

Cannot read property 'nameCapitalized' of undefined
```

Reply:
I think I've realized why it wasn't working - I changed it to `relationship.otherEntity.primaryKey.nameCapitalized` and it seems to be working now. Will push the changes.